packages/Graph/Schemas/Microsoft.Graph.Calendar.Helpers.FilterEvents.schema

```diff
@@ -24,47 +24,95 @@
         "turn.result"
       ]
     },
-    "eventsProperty": {
-      "$ref": "schema:#/definitions/valueExpression",
+    "events": {
+      "$ref": "schema:#/definitions/arrayExpression",
       "title": "Events",
       "description": "Events to filter",
       "examples": [
-        "=turn.eventsfilter"
+        "=turn.eventsToFilter"
+      ],
+      "default": [
+        {
+          "id": "00000000-0000-0000-0000-000000000000",
+          "attendees": [
+            {
+              "EmailAddress": {
+                "Address": "<attendee email>",
+                "Name": "<attendee name>"
+              }
+            }
+          ],
+          "description": "<description>",
+          "end": {
+            "@odata.type": "microsoft.graph.dateTimeTimeZone",
+            "dateTime": "2021-01-01T08:30:00Z",
+            "timeZone": "Pacific Standard Time"
+          },
+          "index": 0,
+          "isAllDay": false,
+          "isCurrentEvent": false,
+          "isOnlineMeeting": false,
+          "isOrganizer": true,
+          "isRecurring": true,
+          "location": "<location>",
+          "organizer": {
+            "@odata.type": "microsoft.graph.recipient",
+            "emailAddress": {
+              "@odata.type": "microsoft.graph.emailAddress",
+              "address": "<organizer email>",
+              "name": "<organizer name>"
+            }
+          },
+          "response": "organizer",
+          "start": {
+            "@odata.type": "microsoft.graph.dateTimeTimeZone",
+            "dateTime": "2021-01-01T08:00:00Z",
+            "timeZone": "Pacific Standard Time"
+          },
+          "subject": "<subject>",
+          "webLink": ""
+        }
       ]
     },
-    "titleProperty": {
-      "$ref": "schema:#/definitions/valueExpression",
+    "title": {
+      "$ref": "schema:#/definitions/stringExpression",
       "title": "Title",
       "description": "Title to filter by",
       "examples": [
         "Example Meeting"
       ]
     },
-    "locationProperty": {
-      "$ref": "schema:#/definitions/valueExpression",
+    "location": {
+      "$ref": "schema:#/definitions/stringExpression",
       "title": "Location",
       "description": "Location to filter by",
       "examples": [
         "34/1880"
       ]
     },
-    "attendeesProperty": {
-      "$ref": "schema:#/definitions/valueExpression",
+    "attendees": {
+      "$ref": "schema:#/definitions/arrayExpression",
       "title": "Attendees",
       "description": "Attendees to filter by",
       "examples": [
-        "[\"janedoe@contoso.com\", \"johndoe@contoso.com\"]"
+        "=turn.attendeesParameter",
+        "[ 'Jane', 'John' ]"
+      ],
+      "default": [
+        "<name to filter by>"
```

Comment:
Nit: should the example be an array of strings instead?

Reply:
You mean an array of empty strings? This will show up in the editor box so I wanted to try an make it obvious it needs to be changed

Reply:
I was going to say sample names but whatever you have works still